hera_cal/abscal.py

```diff
@@ -233,6 +238,8 @@ def phs_logcal(self, avg=False, verbose=True):
 
         Parameters:
         -----------
+        refant : antenna number integer to use as reference antenna, whose phase is set to zero
```

Comment:
You don't actually need this here, since it's not a keyword argument of the function.

Reply:
ah yeah, that was a holdover from when i was allowing it to be passed to each function, thanks for the catch